stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/FileAttachmentContent.kt

```diff
@@ -85,24 +95,30 @@ public fun FileAttachmentContent(
  * Represents each file item in the list of file attachments.
  *
  * @param attachment The file attachment to show.
+ * @param onLongClick Handler used when the item is long clicked.
+ * @param onClick Handler used when the item is clicked.
  */
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
-public fun FileAttachmentItem(attachment: Attachment) {
-    val previewHandlers = ChatTheme.attachmentPreviewHandlers
+public fun FileAttachmentItem(
+    attachment: Attachment,
+    onLongClick: () -> Unit = {},
```

Comment:
Rather than having these two parameters - could we rather expose a `Modifier` and then we could pass in a modifier with a clickable listener?

That way we don't force people to pass in two handlers they might not need to use.